tox.ini

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+requires =
+    tox>=4
+envlist = docs
+
+[testenv]
+basepython = python3
+setenv =
+  PY_COLORS=1
+passenv =
+  GITHUB_TOKEN
+
+[testenv:docs]
```

Comment:
Shouldn't this be called something like `make-prs` (also on line 6), rather than `docs`?